turbopy/diagnostics.py

```diff
@@ -109,6 +110,23 @@ def _write_buffer(self):
             np.savetxt(f, self._buffer, delimiter=",")
 
 
+
+class NetCDFOutputUtility(OutputUtility):
+    """Diagnostic output utility that writes to NetCDF files
+    """
+    def __init__(self, filename, diagnostic_size, **kwargs):
+        self._filename = filename
+        self._buffer = np.zeros(diagnostic_size)
+        self._buffer_index = 0
+
+
```

Comment:
Does the `diagnose` method from `OutputUtility` need to be overridden?

```suggestion
    def diagnose(self, data):
        """Write data to NetCDF file"""
```

Reply:
Yes. The class `NetCDFOutputUtility` is incomplete. I should probably just remove it.

Reply:
I've removed this class. We can add something like it later if desired.